external/storm-hbase/pom.xml

```diff
@@ -1,92 +1,92 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
That's happened because of my IDE. I would recommend adding the POM-Sort plugin to make sure this is auto fixed when the build runs. 

I took a crack at that a few months ago: https://github.com/srotya/srotya-parent/blob/master/pom.xml